pages/linux/sic.md

```diff
@@ -0,0 +1,29 @@
+# sic
+
+> simple irc client.
+> Part of suckless.org tools.
+> More information: <https://tools.suckless.org/sic/>.
+
+- Connect to host (default: irc.oftc.net) with nickname (default: $USER):
+
+`sic -n {{nickname}}`
+
+- Connect to a different host with PASS connection password:
+
+`sic -h {{host}} -n {{nickname}} -k {{password}}`
+
+- Usage with highlighted alerts on mention of your username:
+
+`sic | awk '/username/ {printf "\a"}1'`
```

Comment:
This example seems to demonstrate the usage of the `awk` command, rather than the `sic` command. Generally in tldr pages we try to keep pages focused on the command they are about.

Reply:
No, this example demonstrates how you can highlight a message when your username is mentioned when using `sic`.

Reply:
Yeah, I'm not sure if we should keep this example.